src/routes/info.ts

```diff
@@ -1,23 +1,30 @@
-import express, { Request, Response } from 'express'
-import { Token } from '../../models/token'
+import express, { Request, Response } from 'express';
+import { getAccountBalanceOfToken, getCirculatingSupply, getTokenTotalSupply, getBurnedTokenAmount, getContractSourceCode } from '../services/bsc-scan.service';
+import { getTokenHolders } from '../services/covalent.service';
+import { getHoneyPotInfo } from '../services/honeypot.service';
 
-const router = express.Router()
+const router = express.Router();
+
+const moralisServerUrl: string = process.env.MORALIS_SERVER_URL!;
+const moralisAppId: string = process.env.MORALIS_APP_ID!;
+const moralisSecret: string = process.env.MORALIS_SECRET!;
 
 router.get('/api/info', async (req: Request, res: Response) => {
     // 1.1
     // @TODO -> pass token contract address and chain ID (BSC, ETH, Polygon)
-    const tokenInfo = await Token.find({}) // If no results go to 1.2
-    return res.status(200).send(tokenInfo)
+    return res.status(200).send(await lookForToken(req.body.address));
```

Comment:
Can we change req.body.address to query params instead?

Reply:
Of course, I had that in mind aswell.